fe/fe-core/src/main/java/com/starrocks/common/proc/DbsProcDir.java

```diff
@@ -59,7 +59,7 @@ public class DbsProcDir implements ProcDirInterface {
             .add("LastConsistencyCheckTime").add("ReplicaQuota")
             .build();
 
-    private GlobalStateMgr globalStateMgr;
+    private final GlobalStateMgr globalStateMgr;
```

Comment:
you should give an example in the commit message for the problem and solution.

Reply:
updated